doc/installation.md

```diff
@@ -13,8 +13,7 @@ To install the package for the current user, run this command:
 
 Where `<SOURCE>` is the location of the package source, e.g. a git repo:
 
-    pip3 install --user git+https://github.com/kernelci/kcidb.git
-
+    pip3 install --user git+https://git@github.com/kernelci/kcidb.git@main
 a (release) tag in a git repo:
 
     pip3 install --user git+https://github.com/kernelci/kcidb.git@v9
```

Comment:
This, however, needs to be removed, as it mentions releases.

Reply:
done